README.md

```diff
@@ -1,2 +0,0 @@
-# Hello-Microverse
-Just the first projet in Microverse to make sure one knows basics of github-flow.
```

Comment:
- [ ] Kindly make sure that you add and personalize your ReadMe from the Microverse ReadMe template [here](https://github.com/microverseinc/readme-template). Make sure you keep all the main headers from the template.